app/views/posts/_post.html.erb

```diff
@@ -0,0 +1,7 @@
+<div class='single-user-info'>
+    <h3>Post #<%= index + 1 %></h3>
+    <p><%= post.text%>...</p>
+    <div class='counters'>
+    <span>Comments: <%= post.comments.size ||= 'No comments yet' %></span>,
+    <span>Likes: <%= post.likes.size ||= 0 %></div></span>
```

Comment:
- [ ] [REQUIRED] Kindly use the likes_counter -  `<small>Likes: <%= post.likes_counter || 0 %></small>`